samcli/commands/local/cli_common/options.py

```diff
@@ -48,7 +48,12 @@ def local_common_options(f):
             default=False,
             help="If set, will emulate a shutdown event after the invoke completes, "
             "in order to test extension handling of shutdown behavior.",
-        )
+        ),
+        click.option(
+            "--container-host",
+            default="localhost",
```

Comment:
Since it seems we don't show defaults... by default, should probably include `show_default=True` here.

Reply:
+1 I like this idea.